templates/typescript/samples/sample-assistant/src/bots/defaultActivityHandler.ts

```diff
@@ -94,4 +94,8 @@ export class DefaultActivityHandler<T extends Dialog> extends TeamsActivityHandl
                 break;
         }
     }
+
+    protected async onEndOfConversationActivity(turnContext: TurnContext): Promise<void>{
+        return DialogEx.run(this.dialog, turnContext, this.dialogStateAccessor);
```

Comment:
Following this [update](https://github.com/microsoft/botframework-solutions/blob/next/templates/csharp/VA/VA/Bots/DefaultActivityHandler.cs#L100)

```suggestion
        await DialogEx.run(this.dialog, turnContext, this.dialogStateAccessor);
```